code_folder/19_model_production_load_test.py

```
@@ -0,0 +1,393 @@
+        max_p99_latency_ms=300.0,
+        max_error_rate=0.02,
+    )
+
+    print(f"\n  Target endpoint: {config.endpoint_url}")
+    print(f"  Phases: {len(config.phases)}")
+    print(f"  Thresholds: min_tps={config.min_tps}, "
+          f"p95<={config.max_p95_latency_ms}ms, "
+          f"error_rate<={config.max_error_rate}")
+
+    # -- Run load test ------------------------------------------------------
+    print(f"\n  Starting load test...\n")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)